CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file
 automatically by Versionist. DO NOT EDIT THIS FILE MANUALLY!
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## v6.2.0 - 2018-05-09
+
+* Feat(linux): Add unique device path #278 [User Name]
```

Comment:
LOL, `[User Name]` ???

Reply:
Didn't have my name in the .gitconfig on my Linux box (Oops)